docs/components/motor/_index.md

```diff
@@ -467,7 +467,7 @@ Stepper motors will report `true` if they are being powered while holding a posi
 
 **Returns:**
 
-- [tuple](https://docs.python.org/3/library/stdtypes.html#tuple)[[bool](https://docs.python.org/3/library/functions.html#bool), [float](https://docs.python.org/3/library/functions.html#float)]: The bool is true if the motor is currently running; false if not.

```

Comment:
Why remove the tuple link?

Reply:
Just makes the type text appear more clean. In a few other places consistently like with extra params we haven't been linking to every type, just the most relevant one. I wouldn't be opposed to changing this up if you want but if so I would want to make it consistent across all nested types.

Reply:
I guess I just figure the info's already there and it doesn't hurt to leave it? But don't feel wildly strongly